package.json

```diff
@@ -13,21 +13,31 @@
     "@react-navigation/native": "^6.0.13",
     "@react-navigation/native-stack": "^6.9.0",
     "@react-navigation/stack": "^6.3.0",
-    "react": "18.1.0",
+    "dotenv": "^16.0.2",
+    "react": "^18.1.0",
```

Comment:
add a ^ in front of version might cause some conflict errors. Please read more about this https://docs.npmjs.com/cli/v8/configuring-npm/package-json#dependencies